### PR TITLE
Update DSEthiop6.xml

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Some revisions

### CODE REVIEW

File: DayrSuryan/DSEthiop6.xml [hunk 1]
```diff
@@ -165,141 +174,153 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
                         </item>
                      </list>
                   </additions>
 
                   <bindingDesc>
                      <binding xml:id="binding">
-                        <decoNote xml:id="b1">Non-Ethiopian binding. Red leather cover with 
-                           imprinted decorations (gilded frame and medallion).</decoNote>
+                        <decoNote xml:id="b1">Non-Ethiopian binding. Red leather cover with
+                           imprinted decorations (a gilded frame and a medallion in the center).</decoNote>
```
Comment (thea-m) [inline DayrSuryan/DSEthiop6.xml:298]: centre

File: DayrSuryan/DSEthiop6.xml [hunk 2]
```diff
@@ -165,141 +174,153 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                      and another word was inserted in a secondary hand, to indicate the
+                      name of the new owner <persName ref="PRS11958WaldaM">Walda Māryām</persName>. The name
+                      appears also in a supplication added
+                      to the end of the main work in the same hand.
+                    <persName ref="PRS11958WaldaM"/> might be identical with
+                     ʿAbd Maryam mentioned in <ref target="#a2">the Arabic note Addition 2</ref>
+                     (Gabra Māryām in Ethiopic).
+                       The name of the Ethiopian hermit <persName ref="PRS11940GabraK">ʿAbd al-Masīḥ al-Ḥabašī</persName>
                         is mentioned with a question mark
-                   in <ref target="#a5">Addition 5</ref>, which might be a more recent speculative addition.
```
Comment (thea-m) [inline DayrSuryan/DSEthiop6.xml:296]: It is now a4

Comment (DenisNosnitsin1970) [reply DayrSuryan/DSEthiop6.xml:296]: Ok

File: DayrSuryan/DSEthiop6.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply DayrSuryan/DSEthiop6.xml:298]: ok

**Comment by thea-m** [comment]
regarding centre or center, I thought we followed british english?

**Pull request merged**